Link to 'contributing code' from 'contributing' (#907)
returntocorp/semgrep-docs: docs/contributing/contributing.md

|      Contribution      |      Where to contribute      |
|------------------------|-------------------------------|
| File a Semgrep issue. | See the [Semgrep GitHub repository](https://github.com/returntocorp/semgrep/issues/new/choose). |

> | Contribute code changes. | Follow the [Contributing code](../contributing/contributing-code.md) guide. Find a task to work on on the [list of good first issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). |

Reviewer: ```suggestion
| Contribute code changes. | Follow the [Contributing code](/contributing/contributing-code) document. Find a task in the [list of good first issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). |
```

| Contribute code changes. | Follow the [Contributing code](/contributing/contributing-code) document. Find a task in the [list of good first issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). |
| Contribute rules to the Semgrep Registry. | Add new rules through Semgrep Cloud Platform or GitHub. See [Contributing rules](/contributing/contributing-to-semgrep-rules-repository/). |

> | Add a new feature. | See the [enhancement issues](https://github.com/returntocorp/semgrep/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) for inspiration. |

Reviewer: @underyx why are we removing this line?
